guides/micronaut-oauth2-github/java/src/main/java/example/micronaut/GithubUser.java

```diff
@@ -1,41 +1,31 @@
 package example.micronaut;
 
-import com.fasterxml.jackson.databind.PropertyNamingStrategy;
-import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import io.micronaut.core.annotation.Creator;
 import io.micronaut.core.annotation.Introspected;
 
 @Introspected
 public class GithubUser {
 
-    private String login;
-    private String name;
-    private String email;
-
-    public GithubUser() {
+    private final String login;
+    private final String name;
+    private final String email;
 
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to keep, we need to add a callout and explain it to the user.